planet/models.py

```diff
@@ -265,6 +266,29 @@ def _get_random_filename(content_type=None):
     return name
 
 
+@define
+class OrderComponent:
+    delivery: str
+    expires: datetime
+    location: str
+    name: str
+
+
+@define
+class Order:
+    created: datetime
+    error_hints: typing.List[str]
+    id: str
+    last_message: str
+    last_modified: datetime
+    request: dict
```

Comment:
`request` primarily exists to contain the user-provided order and delivery criteria. In this implementation it will also contain the entire order description sent back by the API.